### PR TITLE
3540 add env file

### PR DESCRIPTION
**Pull request by alibombaye**

### CODE REVIEW

File: app/authProvider.js [hunk 1]
```diff
@@ -1,23 +1,20 @@
...
-      issuer: OIDC_BASE_URI,
-      clientID: OIDC_CLIENT_ID,
-      clientSecret: OIDC_CLIENT_SECRET,
-      authorizationURL: `${OIDC_BASE_URI}/connect/authorize`,
-      userInfoURL: `${OIDC_BASE_URI}/connect/userinfo`,
-      tokenURL: `${OIDC_BASE_URI}/connect/token`,
-      callbackURL: OIDC_REDIRECT_URI,
+
+    this.passport.use(new PassportClient.Strategy({
+      issuer: config.oidcBaseUri,
```
Comment (kat1906) [inline app/authProvider.js]: deconstruct these in the import

File: app/config.js [hunk 2]
```diff
@@ -8,6 +8,9 @@ module.exports = {
   // Port to run local development server on
   port: process.env.PORT || 3000,
 
+  // The base uri the app is running on to pass to identity service for redirection
+  apBaseUri: process.env.APP_BASE_URI || 'http://localhost:3000',
```
Comment (kat1906) [inline app/config.js]: typo apBaseUri

Comment (alibombaye) [reply app/config.js]: 😢

File: server.js [hunk 3]
```diff
@@ -1,9 +1,20 @@
+require('dotenv').config();
 const browserSync = require('browser-sync');
 const config = require('./app/config');
 const { App } = require('./app');
 const { AuthProvider } = require('./app/authProvider');
 const { routes } = require('./app/routes');
 
+Object.keys(config).map((configKey) => {
```
Comment (kat1906) [inline server.js]: can use object.entries

File: server.js [hunk 4]
```diff
@@ -1,9 +1,20 @@
...
+  if (config[configKey]) {
+    // eslint-disable-next-line no-console
+    console.log(`${configKey} set to ${config[configKey]}`);
```
Comment (kat1906) [inline server.js]: use logger?

File: server.js (hunk 3 above)
Comment (alibombaye) [reply server.js]: I tried that initially but doesn't recognise it here

File: server.js (hunk 4 above)
Comment (alibombaye) [reply server.js]: also tried that 😄 but kept saying logger.info is not a function or something

File: server.js (hunk 3 above)
Comment (kat1906) [reply server.js]: interesting

File: app/authProvider.js (hunk 1 above)
Comment (alibombaye) [reply app/authProvider.js]: 👍

File: server.js (hunk 3 above)
Comment (alibombaye) [reply server.js]: probably why we use require here instead of import. babel not transpiling this file?

**Pull request merged**